src/web-vitals-autoinstrumentation.ts

```diff
@@ -379,31 +484,49 @@ export class WebVitalsInstrumentation extends InstrumentationAbstract {
       nextPaintTime,
       presentationDelay,
       processingDuration,
+      longAnimationFrameEntries,
     }: INPAttribution = attribution;
     const attrPrefix = this.getAttrPrefix(name);
-
-    const span = this.tracer.startSpan(name);
-
-    span.setAttributes({
-      ...this.getSharedAttributes(inp),
-      [`${attrPrefix}.input_delay`]: inputDelay,
-      [`${attrPrefix}.interaction_target`]: interactionTarget,
-      [`${attrPrefix}.interaction_time`]: interactionTime,
-      [`${attrPrefix}.interaction_type`]: interactionType,
-      [`${attrPrefix}.load_state`]: loadState,
-      [`${attrPrefix}.next_paint_time`]: nextPaintTime,
-      [`${attrPrefix}.presentation_delay`]: presentationDelay,
-      [`${attrPrefix}.processing_duration`]: processingDuration,
-      // These will be deprecated in a future version
-      [`${attrPrefix}.element`]: interactionTarget,
-      [`${attrPrefix}.event_type`]: interactionType,
-    });
-
-    if (applyCustomAttributes) {
-      applyCustomAttributes(inp, span);
-    }
-
-    span.end();
+    const inpDuration = inputDelay + processingDuration + presentationDelay;
+    this.tracer.startActiveSpan(
+      name,
+      { startTime: interactionTime },
+      (inpSpan) => {
+        const inpAttributes = {
+          ...this.getSharedAttributes(inp),
+          [`${attrPrefix}.input_delay`]: inputDelay,
+          [`${attrPrefix}.interaction_target`]: interactionTarget,
+          [`${attrPrefix}.interaction_time`]: interactionTime,
+          [`${attrPrefix}.interaction_type`]: interactionType,
+          [`${attrPrefix}.load_state`]: loadState,
+          [`${attrPrefix}.next_paint_time`]: nextPaintTime,
+          [`${attrPrefix}.presentation_delay`]: presentationDelay,
+          [`${attrPrefix}.processing_duration`]: processingDuration,
+          [`${attrPrefix}.duration`]: inpDuration,
+          [`${attrPrefix}.timing.json`]: JSON.stringify(
```

Comment:
Is it possible for us to unpack the JSON object here instead of stringifying it? It makes querying this data a lot harder if it is JSON and more attributes are generally ok for us to add.

Reply:
Absolutely.

Reply:
>Absolutely.

This was a bit optimistic. Since this a JSON Array and not a JSON object it doesn't fit cleanly into `SpanAttributes` so I'm going to remove this for now and we can open a follow up PR with this data.